Makefile

```diff
@@ -1,13 +1,13 @@
-clean:
-	# Remove the build
-	rm -rf url.egg-info build dist
-	# And all of our pyc files
-	find . -name '*.pyc' | xargs -n 100 rm
-	# And lastly, .coverage files
-	find . -name .coverage | xargs rm
+.PHONY: test
+test:
```

Comment:
I'm not sure how linting works with Cython -- perhaps we can revisit this after the cython switch.